dotnet-trace/test.sh

```diff
@@ -48,14 +48,14 @@ else
    exit 1
 fi 
 
-dotnet-trace report $FILENAME topN
-if [ -f $REPORTNAME ]; then
+if dotnet-trace report $FILENAME topN | grep -q 'Missing Symbol'; then
```

Comment:
`grep -q` exits immediately if a match is found, which can cause the previous commands in the pipeline to report a SIGPIPE. That leads to non-reproducible and hard to debug issues.